mackup/applications/fisherman.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Fisherman
+
+[configuration_files]
+.config/fisherman
```

Comment:
@sallar sure, it's not late to fix it 👍 

I spent quite some time just now, fighting with google-drive which tried to sync my local plugins `.git` folders 😅 it didn't want to unsync it, and it's thousands of files..